net/initiate.go

```diff
@@ -126,29 +123,35 @@ func (h *Host) handleProtocolStream(stream libp2pnetwork.Stream) {
 		return
 	}
 
-	var s SwapState
-	s, resp, err := h.makerHandler.HandleInitiateMessage(curPeer, im)
-	if err != nil {
-		log.Warnf("failed to handle protocol message: err=%s", err)
+	h.swapMu.Lock()
+	if h.swaps[im.OfferID] != nil {
+		log.Warnf("ignoring attempting initiation of swap %s: %s", im.OfferID, errSwapAlreadyInProgress)
```

Comment:
If I'm understanding correctly what this is checking, I think we want a clearer message. Would it be correct to say something like this:
`ignoring initiation from peerID=%s for offer ID=%s that is already taken`

It is a scenario that will probably happen frequently if the offer is really good.

Reply:
the `errSwapAlreadyInProgress` says that the swap is already ongoing, should i change it to say "offer no longer available" or something like that? and yeah the offer isn't advertised when it's taken so this would only happen if two users try to initiate at basically the same time